src/core/lib/gprpp/ref_counted.h

```diff
@@ -56,7 +56,7 @@ class NonPolymorphicRefCount {
 // e.g., class MyClass : public RefCounted<MyClass>
 //
 // Use PolymorphicRefCount and NonPolymorphicRefCount to select between
-// different implemetnation RefCounted.
+// different implemetnations of RefCounted.
```

Comment:
s/implemetnations/implementations/

Reply:
Thank you for catching this! Sorry I made a stupid typo, not once but twice :P